core/common/src/TextureProps.ts

```diff
@@ -27,9 +27,11 @@ export interface TextureProps extends DefinitionElementProps {
 }
 
 /** Properties that specify what texture should be loaded and how it should be loaded.
- * @alpha
+ * @public
  */
 export interface TextureLoadProps {
   /** A valid Id64 string identifying the texture */
   name: Id64String;
+  /** Maximum texture size supported by the client. The texture will be downsampled so both of its dimensions adhere to this size. */
+  maxTextureSize: number;

```

Comment:
This has to be optional to avoid breaking apps using an older version of imodeljs than the backend is using.

Reply:
Made optional.